ts/src/alpaca.ts

```diff
@@ -61,8 +61,6 @@ export default class alpaca extends Exchange {
                 'fetchFundingHistory': false,
                 'fetchFundingRate': false,
                 'fetchFundingRates': false,
-                'fetchL1OrderBook': true,
-                'fetchL2OrderBook': false,
```

Comment:
why remove the `'fetchL2OrderBook': false,` ?

Reply:
@carlosmiei  Respectfully, this is finishing up one of your commits from last year. On July 26 in d9ea5ca, you removed fetchL1OrderBook ( following up on your commit on Jun 15), and updated fetchOrderBook so it was unified, but didn't remove the fetchL1OrderBook flag, and either: set fetchL2OrderBook to true, or remove fetchL2OrderBook:false entirely, as Exchange.js has had fetchL2OrderBook: true set since July 7 2022.

Reply:
@carlosmiei Ok, but since Exchange.ts:499 is `'fetchL2OrderBook': true,` and the inheritance is `true` not `undefined`, does that still hold where each exchange should be specifically set to `true`?

I was trying to have this be the same as the other exchanges.  It looks like `'fetchL2OrderBook': true,` only appears in btcalpha, blockchaincom, currencycom, and mexc.
`'fetchL2OrderBook': false,` only appears in alpaca and coinbase, and I believe should either be changed to true, or removed.  bkex has `'fetchL2OrderBook': undefined,` but does have `'fetchOrderBook': true,` and appears to have the feature.

I can go through and add it to all the exchanges that don't have it, remove it from these 7, or just change alpaca, coinbase, and bkex.